openprescribing/frontend/management/commands/import_measures.py

```diff
@@ -99,15 +104,23 @@ def parse_options(self, options):
             options['measure_ids'] = [k for k in options['measures']]
 
         # Get months to cover from options.
-        if not options['month'] and not options['end_date']:
+        if not options['month'] and not options['end_date'] \
+           and not options['month_from_prescribing_filename']:
             err = 'You must supply either --month or --end_date '
-            err += 'in the format YYYY-MM-DD. You can also '
+            err += 'in the format YYYY-MM-DD, supply a path to a file which '
```

Comment:
Sorry to be niggly but is this comment missing an "or"? I fear that if we don't fix it it will cause confusion in the future.